### DISCUSSION TITLE
Welcome to AirGuard-iOS Discussions!

### DISCUSSION THREAD

**Discussion by Sn0wfreezeDev** [seemoo-lab/AirGuard-iOS#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We do not have issues activated, since we won't be able to give technical support on the app.  If you discover something that does not work as expected, please start a new discussion. If you can reproduce an issue, you can create a pull request with a test. 

Enjoy using our app and keep safe!

**Comment by hazimeh** [comment]
Am I missing something, or is this repo just a placeholder?

**Reply by Sn0wfreezeDev** [reply]
You can start a new discussion, if you have questions

**Comment by schocolo** [comment]
Apple Findmy kompatibele Maginon Smat Tags werden von AirGuard leider auch mit dem manuellen Scan nicht gefunden. Was tun?

**Reply by Sn0wfreezeDev** [reply]
Sind das diese hier von Aldi? 
https://www.aldi-onlineshop.de/p/smart-tag-2er-set-weiss-1025518/ 

Dann kann ich die bestellen, um das zu testen

**Reply by schocolo** [reply]
Ja genau, von Aldi

**Reply by Sn0wfreezeDev** [reply]
Wir haben die bestellt und konnten das Problem nachvollziehen. Wir haben auch bereits eine Lösung gefunden und im nächsten Update werden sie erkannt. 
Die Tags haben sich anders verhalten, als andere Find My Tracker die nicht von Apple sind. Danke für den Hinweis.

**Reply by schocolo** [reply]
Interessant, das es solche Unterschiede gibt.
Hast Du vielleicht einen Tip zum zurücksetzen auf die Werkseinstellungen? Mit der in der Betriebsanleitung angegebenen Methode klappt das bei mir nicht. 
Ich hatte einen der Tags verliehen gehabt, angeblich hat der Freund ihn nun bei sich abgemeldet, trotzdem kann ich ihn nicht finden, bzw. nicht neu koppeln, da er offenbar nicht weis, das er entkoppelt wurde -und der Freund ist ich mehr in der Stadt.
Übrigens soll das Maginon (Aldi) -Modell baugleich mit einem von Rollei sein, die Betriebsanleitung ist identisch.

**Reply by Sn0wfreezeDev** [reply]
Hallo, 
leider nicht. Das Maginon Modell ist für mich auch neu. Ich habe noch nicht probiert es aus einem Account zu nehmen und zu einem anderen hinzuzufügen.

**Comment by AlienMajik** [comment]
Hey for a future feature could add to be able to detect smart glasses like this project?: https://github.com/yjeanrenaud/yj_nearbyglasses
